content/principles.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Example'
+---
+
+<content>
+
+
+# The GDL development principles
+
+The goal for our developer portal is to enable communities of developers, designers and content providers to reuse software, content and design elements from the GDL in new projects.
+
+By taking this open approach to digital development, we hope to help increase collaboration in the digital development community and avoid duplication of work.
+
+The GDL development principles are built upon the [Principles for Digital Development](https://digitalprinciples.org/), that are nine living guidelines designed to help digital development practitioners integrate established best practices into technology-enabled programs.
+
+## Use Open Standards, Open Source, and Open Innovation
+
+An open approach to digital development can help to increase collaboration in the digital development community and avoid duplicating work that has already been done. Programs can maximize their resources — and ultimately their impact — through open standards, open data, open source technologies and open innovation.
+
+We have built the GDL platform with open source software, and all the content available on the platform is licensed under Creative Commons license.
+
+## Designing with the user
+
+All our development starts with getting to know the people we are designing for through conversation, observation and co-creation. Information gathered through this engagement leads to building, testing and redesigning tools until they effectively meet user needs. By designing with the users, and not for them, you can build digital tools to address better the specific context, culture, behaviours and expectations of the people who will directly interact with the technology. Designing together means partnering with users throughout the project lifecycle, co-creating solutions, and continuously gathering and incorporating users’ feedback.
+
+The GDL team has tested the platform in multiple contexts including countries like Kenya, Ethiopia, Bangladesh, Cambodia and Nepal.
+
+Designing with the user includes focussing on Accessibility, developing the GDL platform based on universal design.
+
+## Privacy and security
+
+Addressing privacy and security in digital development involves careful consideration of which data are collected and how data are acquired, used, stored and shared.
+
+The GDL projects have taken measures to minimize the collection of information and to we protect confidential information and identities of individuals represented in data sets from unauthorized access and manipulation by third parties.
```

Comment:
Kanskje skrive om setningen `the collection of information and to we proctect .....`

Reply:
Enig, jeg skriver om.